analyses/cell-type-neuroblastoma-04/references/consensus-marker-genes.tsv

```diff
@@ -0,0 +1,101 @@
+NBAtlas_label	ensembl_gene_id	gene_symbol	direction	source
```

Comment:
To be honest I'm not super convinced that we need this and think we could just read in the TSV that already exists in the `cell-type-consensus` module when we grab marker genes? Is there another reason you think having this separate file will be beneficial?

Reply:
> Is there another reason you think having this separate file will be beneficial?

Mostly I like that the labels are recoded to match what's in NBAtlas to save some us wrangling this every time we'd have to use them

> Another related note is that you might want to keep only genes that have gene_observed_count == 1

👍